optihood/entities.py

```diff
@@ -23,3 +23,215 @@ class CsvInputFilePathsRelative(_enum.StrEnum):
     storages = "storages.csv"
     stratified_storage = "stratified_storage.csv"
     profiles = "profiles.csv"
+
```

Comment:
These entities are important for another project, where we will automatically be writing a lot of scenario files.